### PR TITLE
add max latency reporting to rate and latency pages

### PR DESCRIPTION
**Pull request by harneetsi**
modify rate and latency pages to report max latency as well
as average latency

Signed-off-by: David Hunt <david.hunt@intel.com>
Signed-off-by: Harneet Singh <harneet.singh@intel.com>

### CODE REVIEW

File: app/pktgen-latency.c [hunk 1]
```diff
@@ -184,7 +184,7 @@ pktgen_page_latency(void)
     unsigned sp;
     char buff[32];
     int display_cnt;
-    uint64_t avg_lat, ticks;
+    uint64_t avg_lat, ticks, max_lat = 0;
```
Comment (KeithWiles) [inline app/pktgen-latency.c]: max_lat is set on line 255 to zero, is setting it here also required?

File: app/pktgen-rate.c [hunk 2]
```diff
@@ -307,7 +307,7 @@ pktgen_page_rate(void)
     unsigned sp;
     char buff[32];
     int display_cnt;
-    uint64_t avg_lat, ticks;
+    uint64_t avg_lat, ticks, max_lat;
```
Comment (KeithWiles) [inline app/pktgen-rate.c:310]: This change does not set max_lat == 0.

**Review by KeithWiles — COMMENTED** [review]
Looks good a couple comments/nits

File: app/pktgen-latency.c (hunk 1 above)
Comment (harneetsi) [reply app/pktgen-latency.c]: not required, good spot ! verified by using debugger mode. I will fix that in the next commit. Thanks Keith.

File: app/pktgen-rate.c (hunk 2 above)
Comment (harneetsi) [reply app/pktgen-rate.c:310]: That's not required as there is no use of max_lat till line 388 and on line 379 we are assigning it a value 0.

**Review by KeithWiles — APPROVED** [review]
LGTM thank you.

**Pull request merged**